packages/components/src/components/as-time-series-widget/as-time-series-widget.tsx

```diff
@@ -433,8 +444,12 @@ export class TimeSeriesWidget {
       </as-histogram-widget>];
   }
 
-  private axisFormatter(value: number): string {
-    return this._formatter(new Date(value));
+  private axisFormatter(value: number | Date): string {
+    return this._formatter(value);
+  }
+
+  private _tooltipFormatter(data: TimeSeriesData): string {
+    return `${data.start}, ${data.value}`;
```

Comment:
You can make multi-line tooltips by returning an array of strings, just in case you think it's better to show a line for the start value and then the value itself below.